package.json

```diff
@@ -104,7 +104,6 @@
     "webpack-cli": "6.0.1"
   },
   "extensionDependencies": [
-    "vincaslt.highlight-matching-tag",
     "wmanth.jar-viewer"
```

Comment:
Is this `jar-viewer` also necessarily required as a dependency either or could it be a nice to have as well?